cmd/root.go

```
@@ -138,18 +138,25 @@ func init() {
-		err := tuiUtils.PrintStyledText("ATMOS")
-		if err != nil {
-			u.LogErrorAndExit(schema.AtmosConfiguration{}, err)
+		if command.Use != "terraform" {
+			err := tuiUtils.PrintStyledText("ATMOS")
+			if err != nil {
+				u.LogErrorAndExit(schema.AtmosConfiguration{}, err)
+			}
+		}
+		// TODO: find a better way to do this if possible
+		if command.Use == "terraform" {
+			oldUsageFunc(command)
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)